src/components/classifier/MultiAnswerClassifier.js

```diff
@@ -0,0 +1,332 @@
+        isQuestionVisible,
+        fullScreenImageSource,
+        fullScreenQuestion,
+        showFullSize
+      } = this.state
+
+      if (isFetching || !isSuccess) {
+          return <OverlaySpinner overrideVisibility={isFetching} />
+      }
+
+      const renderTutorial = () =>
+          <View style={styles.tutorialContainer}>
+              <Tutorial
+                  projectName={project.display_name}
+                  isInitialTutorial={needsTutorial}
+                  tutorial={tutorial}
+                  finishTutorial={() => this.finishTutorial()}
+              />
+          </View>
+
+      const question =
+          <View>
+            <Question
+              question={task.question}
+              workflowID={workflow.id}
+              onPressImage={(src, question) => {
+                this.setState({
+                  showFullSize: true,
+                  fullScreenImageSource: ({uri: src}),
+                  fullScreenQuestion: question
+                })}
+              }
+            />
+            {
+              markdownContainsImage(task.question) ?
+                <Separator style={styles.questionSeparator} />
+              :
+                null
+            }
+          </View>
+
+      const seenThisSession = R.indexOf(subject.id, subjectsSeenThisSession) >= 0
+      const classificationPanel =
+          <View style={styles.classificationPanel}>
+            <ClassificationPanel
+              hasTutorial = { !R.isEmpty(tutorial) }
+              isQuestionVisible = {isQuestionVisible }
+              setQuestionVisibility = { this.setQuestionVisibility() }
+            >
+              {
+                isQuestionVisible &&
+                  <View>
+                    { question }
+                  </View>
+              }
+            </ClassificationPanel>
+            {
+              isQuestionVisible ?
+                <ScrollView
+                  style={styles.scrollView}
+                  ref={ref => this.scrollView = ref}
+                >
+                  <View style={styles.backgroundView} />
+                  <View style={styles.classifierContainer}>
+                    <View onLayout={({nativeEvent}) => this.setState({imageDimensions: {width: nativeEvent.layout.width, height: nativeEvent.layout.height}})}>
+                      <TapableSubject
+                        height={300}
+                        width={imageDimensions.width}
+                        subject={subject}
+                        alreadySeen={seenThisSession}
+                        onPress={(imageSource) => this.setState({
+                          showFullSize: true,
+                          fullScreenImageSource: {uri: imageSource}})}
+                      />
+                    </View>
+                    {
+                      answers.map((answer, index) =>
+                        <View key={index} style={styles.buttonContainer}>
+                          <ClassifierButton
```

Comment:
This component isn't rendering Markdown, as the question text is through the `Question` component. Some project builders will put Markdown in their answers, so there should probably be a Markdown wrapper somewhere in this component, perhaps the same `SizedMarkdown` component that `Question` uses.

Reply:
Interesting! Looks like that's a homemade component. I'm trying to incorporate that at the level of `ClassifierButton`, but I'm running into some issues. Would love to pair with you on this.